lib/App/index.js

```diff
@@ -365,6 +365,39 @@ class App {
             });
           }
         }
+
+        if (driver.connectivity && driver.connectivity.includes('matter')) {
+          if (!driver.matter) {
+            throw new Error(`drivers.${driver.id} invalid 'connectivity': 'matter' requires a 'matter' object.`);
+          }
+
+          if (driver.pair) {
+            throw new Error(`drivers.${driver.id} invalid 'pair': matter drivers do not support custom pairing views.`);
+          }
+
+          if (
+            (driver.matter.deviceVendorId === undefined && driver.matter.deviceProductName !== undefined) ||
+            (driver.matter.deviceVendorId !== undefined && driver.matter.deviceProductName === undefined)
+          ) {
```

Comment:
Kunnen we hier niet gewoon checken op `if (driver.matter.deviceVendorId === undefined || driver.matter.deviceProductName === undefined)` als ze toch altijd allebei gedefinieerd moeten zijn?

Reply:
Deze waardes zijn optioneel (alleen voor bridged matter apparaten). Dus ze moeten allebei undefined zijn of allebei een waarde hebben.